package.json

```diff
@@ -41,6 +41,7 @@
     "react-scripts": "5.0.0",
     "react-shimmer-effect": "^1.0.9",
     "react-simple-maps": "^2.3.0",
+    "react-toastify": "^9.0.8",
```

Comment:
Isn't there a toast message lib in already? Please check before adding.

Reply:
Yes, one more package already exists